whois-rdap/src/main/java/net/ripe/db/whois/rdap/RdapLookupService.java

```diff
@@ -219,8 +217,7 @@ private Object getRdapObject(final HttpServletRequest request, final Iterable<Rp
         RpslObject resultObject = rpslIterator.next();
 
         if (rpslIterator.hasNext()) {
-            throw new RdapException("Internal Error", "Unexpected result size: " + Iterators.size(rpslIterator),
```

Comment:
Any idea why the reported size was "1" and not the actual size ? Was it because the `rpslIterator` was already read once, and one remaining?

For example: https://rdap.db.ripe.net/entity/Nada
Should report 2 matches, not one.